src/config.rs

```diff
@@ -23,6 +31,9 @@ impl Config {
             port: default_env("PORT", 3001)?,
             gh_rate_limit_tokens: require_env("RATE_LIMIT_TOKENS")?,
             gh_rate_limit_stats_cache_refresh: default_env("GH_RATE_LIMIT_STATS_REFRESH", 120)?,
+            rust_runners_token: require_env("RUST_ORG_TOKEN")?,
```

Comment:
I'd just call this `GITHUB_TOKEN`, as I'd expect to use it for all the GitHub API requests except for the rate limit monitoring ones.

Reply:
renamed to `GITHUB_TOKEN`